### ISSUE TITLE
Clicking any of the 3 buttons on the Vertipaq Analyzer does not receive any response

### ISSUE THREAD

**Issue by marcussyliu** [sql-bi/VertiPaq-Analyzer#41]
Hi Marco,

When I clicked on Open VPAX button, nothing happened. Same are with the other two buttons on the Vertipaq Analzyer tab.  Actually, I have a similar issue with Analzye in Excel in SSAS tabular project from within SSDT. Clicking the Analzye in Excel button does not trigger any action. Can you please help understand what happened and how to fix the issue?

Many thanks in advance.

**Comment by marcosqlbi** [comment]
If you open the "Open VPAX" button in the Excel file of VertiPaq Analyzer, make sure you enabled macros; otherwise, with macro disabled, you don't see anything happen.
Regarding Analyze in Excel in SSDT, it could be the lack of updated OLE DB drivers, but you should check with Microsoft support for that.

**Comment by marcussyliu** [comment]
Thanks Marco for your prompt response.

I do have enabled macro and allowed access to VBA project i the trust center settings. Also a FYI, I am using Office 365 with the updated version.

**Comment by marcosqlbi** [comment]
I also use the latest version of Office 365, and it works. When you click "Open VPAX", you should see a dialog box open asking for the file name. You could try to open the VBA editor, set a breakpoint on openVpax function, and/or try to run it manually to investigate on what is going on.

**Comment by marcussyliu** [comment]
Thanks Marco for your advice. I will do additioal testing  and feed you back if i have any finding. Thanks again for your time.

**Comment by marcussyliu** [comment]
Here is my finding - after I enabled Excel 4.0 macro when VBA macros are enabled, the problem was fixed. Thanks again for your help.

**Comment by marcosqlbi** [comment]
This is strange - I don't have that setting enabled. Please, can you provide the exact version of Excel you are using? (File / Account / About Excel)

**Comment by marcussyliu** [comment]
it is version 2207 build 16.0.15427.20182.

**Comment by marcosqlbi** [comment]
Did you see the message below when you opened the file you downloaded? Perhaps, you didn't enable the macro at that point. and the macro was kept disabled for that file until you changed a setting. The file doesn't use Excel 4 macros...

![image](https://user-images.githubusercontent.com/13840816/187024609-accc19e1-247d-4ec4-a8fc-5dec705ae8f9.png)

**Comment by marcussyliu** [comment]
Yes I did see these two messages. What I did was I first enabled content and then closed the bar message below. I cannot remember if i had the Excel 4.0 macro setting enabled before or after i found the issue , but if deselect it,  the button gives no response.

**Comment by marcosqlbi** [comment]
Please, let me know if the problem also exists in version 2208 (when you will get the Excel upgrade). I cannot reproduce with 2207.

**Comment by marcussyliu** [comment]
Yes I did see these two messages. What I did was I first enabled content and then closed the bar message below. I cannot remember if i had the Excel 4.0 macro setting enabled before or after i found the issue , but if deselect it,  the button gives no response.
